drivers/applications/mysql/mysql.go

```diff
@@ -0,0 +1,244 @@
+	log.InfoD("Inserting data")
+	if len(commnads) == 0 {
+		log.InfoD("Inserting below data : %s", strings.Join(app.SQLCommands[identifier]["insert"], "\n"))
+		err = app.ExecuteCommand(app.SQLCommands[identifier]["insert"], ctx)
+	} else {
+		log.InfoD("Inserting below data : %s", strings.Join(commnads, "\n"))
+		err = app.ExecuteCommand(commnads, ctx)
+	}
+
+	return err
+}
+
+// Return data inserted before backup
+func (app *MySqlConfig) GetBackupData(identifier string) []string {
+	if _, ok := app.SQLCommands[identifier]; ok {
+		return app.SQLCommands[identifier]["select"]
+	} else {
+		return nil
+	}
+}
+
+// CheckDataPresent checks if the mentioned entry is present or not in the database
+func (app *MySqlConfig) CheckDataPresent(selectQueries []string, ctx context.Context) error {
+
+	log.InfoD("Running Select Queries")
+
+	conn, err := app.GetConnection(ctx)
+	if err != nil {
+		return err
+	}
+
+	defer conn.Close()
+
+	var key string
+	var value string
+	var queryNotFoundList []string
+
+	for _, eachQuery := range selectQueries {
+		currentRow := conn.QueryRowContext(ctx, eachQuery)
+		err := currentRow.Scan(&key, &value)
+
+		if err != nil {
+			log.InfoD("Select query failed - [%s] Error - [%s]", eachQuery, err.Error())
+			queryNotFoundList = append(queryNotFoundList, eachQuery)
+		}
+	}
+
+	if len(queryNotFoundList) != 0 {
+		errorMessage := strings.Join(queryNotFoundList, "\n")
+		return fmt.Errorf("Below results not found in the table:\n %s", errorMessage)
+	}
+
+	return nil
+}
+
+// UpdateBackupData updates the rows generated initially by utilities
+func (app *MySqlConfig) UpdateBackupData(ctx context.Context) error {
+
+	log.InfoD("Running Update Queries")
+	err := app.ExecuteCommand(app.SQLCommands["default"]["update"], ctx)
+
+	return err
+}
+
+// DeleteBackupData deletes the rows generated initially by utilities
+func (app *MySqlConfig) DeleteBackupData(ctx context.Context) error {
+
+	log.InfoD("Running Delete Queries")
+	err := app.ExecuteCommand(app.SQLCommands["default"]["delete"], ctx)
+
+	return err
+}
+
+// StartData - Go routine to run parallal with app to keep injecting data every 2 seconds
+func (app *MySqlConfig) StartData(command <-chan string, ctx context.Context) error {
+	var status = "Start"
+	var allSelectCommands []string
+	var allErrors []string
+	var tableName = "table_" + RandomString(4)
+
+	createTableQuery := fmt.Sprintf(`CREATE TABLE IF NOT EXISTS %s (
+		`+"`key` "+`VARCHAR(45) NOT NULL,
+		value VARCHAR(255)
+	  )`, tableName)
+	err := app.ExecuteCommand([]string{createTableQuery}, ctx)
+	if err != nil {
+		log.InfoD("Error while creating table - [%s]", err.Error())
+		allErrors = append(allErrors, err.Error())
+	}
+	for {
+		select {
+		case cmd := <-command:
+			switch cmd {
+			case "Stop":
+				if len(allErrors) != 0 {
+					return fmt.Errorf(strings.Join(allErrors, "\n"))
+				}
+				err := app.CheckDataPresent(allSelectCommands, ctx)
+				return err
+
+			case "Pause":
+				status = "Pause"
```

Comment:
Make these strings as const of a certain type

Reply:
Done